code/modules/tents/folded_tents.dm

```diff
@@ -144,6 +144,33 @@
 	off_x = -2
 	template = /datum/map_template/tent/big
 
+/obj/item/folded_tent/corp
+	name = "folded company Tent"
+	icon_state = "cmd"
+	desc = "A company liaison tent, provided with all the resources for company field work, and a luxurious rug. ENTRANCE TO THE SOUTH."
+	dim_x = 3
+	dim_y = 4
+	off_x = -2
+	template = /datum/map_template/tent/corp
+
+/obj/item/folded_tent/mp
+	name = "folded MP Tent"
```

Comment:
i would suggest to not use "MP" in the name. call it detention tent or something